web/themes/custom/fluffiness/css/style.css

```diff
@@ -0,0 +1,150 @@
+@import "base/variables.css";
+@import "layout/breakpoints.css";
+/*general*/
+
+*, *:after, *:before {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1, h2, h3, p, a {
+  margin: 0;
+  padding: 0;
+  font-family: var(--main-font);
+  font-weight: var(--regular);
+}
+
+img {
+  width: 100%;
+  max-width: 100%;
+  height: auto;
+}
+
+/*main*/
+
+.container {
+  padding: 16px 20px;
+}
+
+/*.main-container {*/
+/*  display: flex;*/
+/*  justify-content: space-between;*/
+/*  width: 100%;*/
+/*}*/
+
+.layout-content {
+  width: 100%;
+}
+
+/*.layout-sidebar {*/
+/*  background-color: var(--light-grey);*/
+/*  padding: 24px;*/
+/*  max-height: 300px;*/
+/*  margin-left: 32px;*/
+/*  width: 30%;*/
+/*}*/
+
+.layout-header {
+  border: 1px solid #E8EDEE;
+}
+
+.block-system-branding-block {
+  width: 233px;
+}
+
+footer {
+  margin-top: 96px;
+  background-color: var(--light-grey);
+}
+
+/*article teaser*/
+
+.article-teaser__container, .hospital-teaser__container {
+  max-width: 824px;
+}
+
+.article-teaser__image .field__label {
+  display: none;
+}
+
+.article-teaser .node__links {
```

Comment:
if we don't want to display this field, you can hide it also on the Manage display, so this shouldn't be needed
move it to the **Disabled** section